cmake/modules/FindDEAL_II_BOOST.cmake

```diff
@@ -53,6 +53,7 @@ endif()
 # and https://lists.boost.org/Archives/boost/2019/02/245016.php
 set(Boost_NO_BOOST_CMAKE ON)
 
+set(Boost_NO_WARN_NEW_VERSIONS TRUE)
```

Comment:
TRUE, ON, or YES? :-)

Reply:
You can also CamelCase everything I guess :-)